robocopkaka/bootcamp-29-project
# Fix handle errors during login

#### What does this PR do?
Create actions that are dispatched when a login attempt fails or is loading
#### Description of Task to be completed?
* Create an action that's dispatched when a login attempt is failed
* Create an action that's dispatched when a login attempt is loading
* Add a case for handling failed login attempts
* Add a case for handling loading login attempts
* Use toasts to show the user the state of their login attempts
#### How should this be manually tested?
When a user tries to login, they should see a toast showing them the result of their login attempt
#### Any background context you want to provide?
Wasn't dispatching actions for failed login attempts
#### What are the relevant pivotal tracker stories?(if applicable)
#155875651
